.github/workflows/build-kind.yaml

```diff
@@ -32,6 +32,10 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: 1.22.x
+      - name: Setup Kind
```

Comment:
isn't this action to build a kind image? I don't understand why do we now setup a cluster?

Reply:
To have the `kind` binary on the runner.